docs/releasing.md

```diff
@@ -24,6 +24,13 @@ Creating a federation v2 release involves the following steps:
    3. `kubefed2 version` (check that the output is as expected)
    4. `tar cvzf kubefed2.tar.gz kubefed2`
    5. `sha256sum kubefed2.tar.gz > kubefed2.tar.gz.sha`
+4. Package the helm chart for release
+   1.  Adjust the default image tag in values.yaml
+   2.  Update the chart version in Chart.yaml
+   3.  Package the chart using `helm package federation-v2`
+   4.  Place the release packaged chart (.tgz file) in the federation-v2/ folder.
+   5.  Add the new version to the chart index `helm repo index federation-v2/ --url=https://raw.githubusercontent.com/kubernetes-sigs/federation-v2/master/charts/federation-v2/`
+   6.  Ensure index.yaml containes the latest release within the entries array.
 4. Create github release
    - Copy text from old release and replace old tag references
-   - Add `kubefed2.tar.gz` and `kubefed2.tar.gz.sha`
+   - Add `kubefed2.tar.gz` and `kubefed2.tar.gz.sha`
```

Comment:
This appears to be an unrelated formatting change.  Also, an additional line instructing the inclusion of `federation-v2-chart.tar.gz` and `federation-v2-chart.tar.gz.sha` should be added.

Reply:
Unknown how that occurred...
Added: `   - Add `federation-v2-<x.x.x>.tgz` and `federation-v2-<x.x.x>.tgz.sha`